horreum-api/src/main/java/io/hyperfoil/tools/horreum/api/services/DatasetService.java

```diff
@@ -127,22 +127,20 @@ class DatasetSummary {
       @JsonProperty(required = true)
       @Schema(type = SchemaType.INTEGER, implementation = Long.class,
               description = "Dataset Start timestamp", example = "1698013206000")
-      public long start;
+      public Instant start;
       @JsonProperty(required = true)
       @Schema(type = SchemaType.INTEGER, implementation = Long.class,
               description = "Dataset Stop timestamp", example = "1698013206000")
-      public long stop;
+      public Instant stop;
       @Schema(required = true, description="Name of the team that owns the test. Users must belong to the team that owns a test to make modifications",
               example="performance-team")
       public String owner;
       @NotNull
       @JsonProperty(required = true)
-//      TODO:: https://github.com/Hyperfoil/Horreum/issues/801
-//      @Schema( type = SchemaType.INTEGER, implementation = Access.class,
-//              description = "Access rights for the test. This defines the visibility of the Test in the UI",
-//              example = "0")
-      @Schema(required = true, implementation = Access.class)
-      public int access;
+      @Schema( type = SchemaType.INTEGER, implementation = Access.class,
+              description = "Access rights for the test. This defines the visibility of the Test in the UI",
+              example = "0")
+      public Access access;
```

Comment:
See my previous comment about return type here, I think it should return a String for Access so that it it meaningful for users